app/scripts/models/user.js

```diff
@@ -81,6 +83,30 @@ define(function (require, exports, module) {
       return this._storage.get('accounts') || {};
     },
 
+    initializeDep() {
```

Comment:
@kritisingh1 where is this new function called ?

Reply:
I think I need to make another function which calls initializeDep() in user.js itself. I am so sorry I didn't notice this.

Reply:
Actually you don't need this extra function

Reply:
See https://github.com/mozilla/fxa-content-server/pull/5473/files#r138177383